### ISSUE TITLE
Improper sentence framing is observed in FAQs 

### ISSUE THREAD

**Issue by KolliAnitha** [oppia/oppia-android#4527]
**Describe the bug**
Improper sentence framing is observed in FAQs 

**To Reproduce**
1. Go to 'Home' - Click on menu - Select Help
2. Click on 'FAQs'
3. Select How can I change my email/phone number?
4. See steps to change phone number

**Expected behavior**
'If you want to change your phone number' should be seen instead of 'If changing your phone number'

**Demonstration**
![Screenshot_2022-08-24-19-40-59-85_943a62cb4c6fb83e010e1c2e82766a17.jpg](https://user-images.githubusercontent.com/101634267/186442052-a7f9cbc4-5dbf-4cb8-8837-6a288b675bbd.jpg)

**Environment**
Device name: One plus Nord2 5G
Android version : Android 11
App version : 0.8-alpha-091b45a188

**Comment by pratyaksh1610** [comment]
@KolliAnitha 
I would like to work on this issue, 
Please assign this issue to me.
Thanks.

**Comment by BenHenning** [comment]
I think we need to consider reworking the entire FAQ content since this specific point (& likely others) don't even apply to the current version of the app.

@seanlip sending this over to you since this is likely something that the product team would be interested in tackling.

**Comment by BenHenning** [comment]
@pratyaksh1610 unfortunately I don't think this issue will involve an easy fix as it will depend on the product team following up with how we should be revising the FAQ. I suggest looking at other open issues, instead, and maybe circling back once we have a clearer product plan here. :)

**Comment by seanlip** [comment]
Added to backlog. However: @BenHenning, for now, could you please remove the parts of the FAQ that are inaccurate? I'm fine with showing fewer Q&A than expected but I don't think our FAQ should be actively wrong.

**Comment by pratyaksh1610** [comment]
In this my approach was to look for a particular `faq answer` and update it in strings, and add a suitable test for it.

**Comment by seanlip** [comment]
@pratyaksh1610 To clarify, I think Ben is saying that this issue cannot be worked on because we need to clarify its definition before deciding to fix it. The technical part is not the problem.

Please consider picking another issue. You might like to check out https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#finding-something-to-do for a list of starter projects.

**Comment by seanlip** [comment]
This is blocked on design: https://github.com/oppia/design-team/issues/75
